students/idcrickmore/lesson05/mailroom_pt3.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+from operator import itemgetter
+import os
+import datetime
+
+
+def menu():
+
+    print("""\n------------------ MENU ------------------\n
+PLEASE CHOOSE FROM THE FOLLOWING THREE OPTIONS
+1. Send Thank You
+2. Export Letters to Everyone
+3. Create a Report
+4. Quit\n""")
+
+    choice = input("-> ").lower()[0]
+
+    # use .get() on the 'options' dict to call functions
+    # keep calling the 'menu' function if user input not in dict
+    options.get(choice, menu)()
```

Comment:
should have an exception handling here if user enters invalid choice.

i would recommend accessing by name instead of via get() to catch `KeyError`